storage/src/lib.rs

```diff
@@ -15,6 +15,8 @@ use s3::{creds::error::CredentialsError, error::S3Error, Bucket};
 pub use s3::{creds::Credentials, Region};
 use serde::Deserialize;
 
+pub const STORAGE_ENDPOINT: &str = "http://localhost:9000";
```

Comment:
Same question as above with the kafka const. Maybe I'm not understanding the use case for this PR fully?